app/models/manageiq/providers/nuage/network_manager/vsd_client.rb

```diff
@@ -0,0 +1,159 @@
+require 'rest-client'
+require 'rubygems'
+require 'json'
+module ManageIQ::Providers
+  class Nuage::NetworkManager::VsdClient
+    def initialize(server, user, password)
+      @server = server
+      @user = user
+      @password = password
+      @rest_call = Rest.new(server, user, password)
+      _is_conn, data = @rest_call.login
```

Comment:
_ in ruby is usually used for non-used parameter/variable. So not as the Python usecase